src/config.rs

```diff
@@ -178,30 +179,38 @@ impl fmt::Display for Config {
 impl Config {
     pub fn maybe_parsed_from_env() -> io::Result<Config> {
         if let Ok(p) = env::var("KEYFILE") {
-            Config::parse_file(p)
+            Config::parse_keyfile(p)
         } else {
             Ok(Config::default())
         }
     }
 
-    pub fn parse_file(p: String) -> io::Result<Config> {
+    pub fn parse_keyfile(p: String) -> io::Result<Config> {
         let mut buf = String::new();
         let mut f = File::open(p)?;
         f.read_to_string(&mut buf)?;
         let mut config = Config::default();
-        for (line_number, line) in buf.lines().enumerate() {
-            let e = format!("invalid config at line {}: {}", line_number, line);
+        for (mut line_num, line) in buf.lines().enumerate() {
+            if line == "" || line.starts_with('#') {
+                continue;
+            }
+
+            // zero based indexing inappropriate here
+            line_num += 1;
+            // TODO Separate error messages to be more informative in each of the two cases
+            // below.
```

Comment:
please either implement the item in the `TODO` or open an issue that refers to this function. I'd prefer not to commit `TODO`'s